src/spikeinterface/sorters/external/kilosort4.py

```diff
@@ -207,7 +210,15 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
         results_dir = sorter_output_folder
         filename, data_dir, results_dir, probe = set_files(settings, filename, probe, probe_name, data_dir, results_dir)
         if version.parse(cls.get_sorter_version()) >= version.parse("4.0.12"):
-            ops = initialize_ops(settings, probe, recording.get_dtype(), do_CAR, invert_sign, device, False)
+            ops = initialize_ops(
+                settings,
+                probe,
+                recording.get_dtype(),
+                do_CAR,
+                invert_sign,
+                device,
+                save_preprocesed_copy=save_preprocessed_copy,  # this kwarg is correct (typo)
```

Comment:
No way..... I have a feeling is is going to be fixed and then we will need another version just to account for this....

Reply:
@zm711 you need to reviow KS PRs ahahahah

Reply:
@JoeZiminski maybe we have to go back to positional arguments here? :)

Reply:
Yes I think so @alejoe91 or the number of conditionals will explode, shall we merge this and then continue the discussion on how best to proceed [here](https://github.com/SpikeInterface/spikeinterface/pull/3316#issuecomment-2298683709)?

Reply:
Let's do that!